homework-submission/week3_app/debug.log

```diff
@@ -0,0 +1,69 @@
+[1016/142102.842:ERROR:file_io_win.cc(179)] CreateFile C:\Program Files (x86)\settings.dat: Toegang geweigerd. (0x5)
```

Comment:
Log files shouldn't be commited

Reply:
I opened the repository folder in VScode. then in homework folder I created ReactAap. In this case I have 2 git files. The first one belongs to the repository, the second one belongs to the ReactApp.
it is a bit confusing. So what is the best thing to do in that case?